src/Core_Command.php

```diff
@@ -158,6 +161,20 @@ public function download( $args, $assoc_args ) {
 			$download_url = str_replace( '.zip', '.tar.gz', $offer['download'] );
 		}
 
+		if ( true === \WP_CLI\Utils\get_flag_value( $assoc_args, 'skip-content' ) && 'en_US' !== $locale ) {
+			WP_CLI::error( 'The skip content build is only available for the en_US locale.' );
+		}
+
+		if ( true === \WP_CLI\Utils\get_flag_value( $assoc_args, 'skip-content' ) && isset( $assoc_args['version'] ) ) {
+			WP_CLI::error( 'The skip content build is only available for the latest version.' );
+		}
+
+		if ( true === \WP_CLI\Utils\get_flag_value( $assoc_args, 'skip-content' ) ) {
+			$offer = unserialize( self::_read( 'https://api.wordpress.org/core/version-check/1.6/' ) );
+			$download_url = $offer['offers'][0]["packages"]["no_content"];
```

Comment:
We need to check that `$offer['offers'][0]["packages"]["no_content"];` is set, and error if it's not.

Reply:
Thanks for your feedback, I've just addressed this!